### ISSUE TITLE
Is there a Central FAQ Issue for common newbie questions?

### ISSUE THREAD

**Issue by kivo360** [RediSearch/RediSearch#1210]
I'm new to this. A lot of people are. While the docs explain a lot, finding everything inside of the docs isn't entirely easy.

I personally had a question about **_how to do an aggregate search query between two keys and schemas_**. I feel like this is a rather straightforward question; one that's been discussed out before. I can't help but notice many questions of this level of difficulty are being asked on a regular basis. For example:

This question here asks how [to do an equal query in redisearch](https://github.com/RediSearch/RediSearch/issues/1206).

I feel like forcing people to ask common questions like this through an entirely different issue reduces attention payed to bigger issues people would have. 

This isn't a criticism. I believe RediSearch will be orders of magnitudes more valuable now that RedisGraph has it integrated into it. In the event other major modules are integrated in, I can see the Redis ecosystem spiking in value. Modules need to be easier to work with to increase the intrinsic value. 

A part of adoption requires that people are able to find answers to common questions. I'm asking a question. Is there a way to make finding and asking frequently answered questions easier in a way that it won't subtract against serious bugs people face?

Perhaps:

1. Having a central FAQ issue people will post common questions to?
2. Reposting responses to stackoverflow and internally upvoting the answers so they can be found using Google.
3. Maybe using Ray's issue scheme, where people are forced to set an issue type like seen below.

![image](https://user-images.githubusercontent.com/2403240/80268225-6e69a600-8673-11ea-8b1f-a2dd23fa7a5f.png)

My real question: **_how to do an aggregate search query between two keys and schemas_**. Even still, I wanted to ask a meta question just out of curiosity.

**Comment by MeirShpilraien** [comment]
@kivo360 thank you very much, your feedback is important to us. I agree that a FAQ is a very good idea. Just FYI we also have a forum specifically for questions and discussion which are not bugs or issues (https://forum.redislabs.com/c/modules/64).
Regarding your question, so currently it is not directly supported by RediSearch. RediSearch allows to query and aggregate results from only one index. But you can do it with RedisGears (https://github.com/RedisGears/RedisGears). What you need to do is to write a python reader (https://oss.redislabs.com/redisgears/master/readers.html#pythonreader) that performs the queries on the indexes and yield the results, and then you just need to use this reader to create Gears function (https://oss.redislabs.com/redisgears/master/functions.html) and aggregate the data. If you need a reference you can look at how the KeysOnlyReader is implemented (https://github.com/RedisGears/RedisGears/blob/b1f9e9369dff478c1895ed5e8e5ffa5feec61b99/src/GearsBuilder.py#L18) it should be very similar.

Let me know if you need any more help, we and we can continue the discussion on RedisGears forum (https://forum.redislabs.com/c/modules/redisgears/59).

**Comment by kivo360** [comment]
@MeirShpilraien Thanks! I happened to see RedisGears just too days ago and I already gave it a try. The register function will be super useful for an internal tool for an internal tool.

I'm working to finish a feature for a project, and I'll likely use an inefficient method so it can be done for now. 


I'm planning a refractor for the internal tool that'll likely solve the aggregate problem too. RedisGears is going to be the key for this.

Thank you for doing awesome work!

I'm gonna be blowing that forum up!
